FireRender.Maya.Src/Context/TahoeContext.cpp

```diff
@@ -514,3 +514,19 @@ void TahoeContext::AbortRender()
 		GetScope().Context().AbortRender();
 	}
 }
+
+void TahoeContext::SetupPreviewMode()
+{
+	if (m_PluginVersion == TahoePluginVersion::RPR1)
+	{
+		RenderType renderType = GetRenderType();
+		int preview = 0;
+
+		if ((renderType == RenderType::ViewportRender) ||
+			(renderType == RenderType::IPR) ||
+			(renderType == RenderType::Thumbnail))
+		{
+			SetPreviewMode(1);
```

Comment:
Is passing constant 1 here correct? Do we never want to pass 0? I'm wondering because in previous implementation in void FireRenderContext::setPreview() both 1 and 0 could be passed to RPR_CONTEXT_PREVIEW